setup.py

```diff
@@ -8,15 +8,22 @@
     long_description = fp.read()
 
 setup(
-    name='Gibberish',
+    name='gibberish',
     description="A pseudo-word generator",
-    version='0.2',
-    author='Gregory Haskins',
-    author_email='greg@greghaskins.com',
-    url='https://github.com/greghaskins/gibberish',
+    version='0.3',
+    author='Alex Damian',
```

Comment:
I'd put 'YPlan' here to avoid issues

you can even use the 'maintainer' field to keep Gregory in there, see: https://github.com/YPlan/gargoyle/blob/master/setup.py#L43